vita3k/gui/src/app_context_menu.cpp

```diff
@@ -97,6 +98,57 @@ static const char OS_PREFIX[] = "open ";
 static const char OS_PREFIX[] = "xdg-open ";
 #endif
 
+static std::map<std::string, std::map<std::string, time_t>> last_time_apps_used;
```

Comment:
Should move it to UI State, then access it there

Reply:
I want keep it here for moment

Reply:
done, moved in gui state.